core/samples/subscribe_event_engine_sample.c

```diff
@@ -0,0 +1,28 @@
+/* -*- c-file-style:"stroustrup"; indent-tabs-mode: nil -*- */
```

Comment:
Have you used sanitizers to test it? 

If not I can do it for you tomorrow (or any other day).